certora/specs/EmergencyMode.spec

```diff
@@ -47,7 +47,8 @@ rule accountCanOnlyLeaveInEmergencyMode(method f) {
   f@withrevert(e, args);
   bool isReverted = lastReverted;
 
-  assert !isReverted => isViewFunction(f) ||
+  assert !isReverted => f.selector == sig:streamer.leave().selector ||
+                        isViewFunction(f) ||
```

Comment:
`leave()` will not revert but should still be protected by a rule that ensures that, if `leave()` was called and the accounting was updated, the stake manager was **not** trusted by the caller (vault).

If it **was** trusted and the vault still manages to leave the system, this is a bug. Vault should properly unstake in that case.